### PR TITLE
Util/validator

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 1]
```diff
@@ -83,6 +84,7 @@ public void getStatusCodeAsResponse(int statusCode) {
      */
     @And("I verify the response schema whit {string}")
     public void iGetASchema(String schema) {
```
Comment (raulchoque-eng) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java]: Put a name more descriptive of parameter schema please

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 2]
```diff
@@ -83,6 +84,7 @@ public void getStatusCodeAsResponse(int statusCode) {
...
-        SchemaValidator.validator(response, "schema/" + schema + "Schema.json");
+        boolean validator = SchemaValidator.validator(response, "schema/" + schema + "Schema.json");
```
Comment (raulchoque-eng) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java]: The concat ("schema/" + schema + "Schema.json") of this string bring to validator method please

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java]: I fixed it.

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java]: I fixed it.

File: src/main/java/com/foundation/trello/util/SchemaValidator.java [hunk 3]
```diff
@@ -6,21 +6,33 @@
 
 /**
  * This class validates the schema of a request response.
+ *
  * @author Josue Rodriguez
```
Comment (MadayAlcala) [inline src/main/java/com/foundation/trello/util/SchemaValidator.java:10]: Please add the version.

Comment (JosueRodriguezGarcia) [reply src/main/java/com/foundation/trello/util/SchemaValidator.java:10]: I fixed it.

File: src/test/java/com/foundation/trello/step/ManagerStepDef.java [hunk 4]
```diff
@@ -20,8 +20,10 @@
 public class ManagerStepDef {
     private RequestManagerAbstract requestManager;
     private Response response;
+    private SoftAssert softAssert;
     private Context context;
```
Comment (MadayAlcala) [inline src/test/java/com/foundation/trello/step/ManagerStepDef.java:25]: Please delete the unnecessary line.

Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/step/ManagerStepDef.java:25]: ok.

**Review by MadayAlcala — APPROVED** [review]

**Review by raulchoque-eng — APPROVED** [review]

**Pull request merged**